### PR TITLE
Add rating breakdown in details view

### PR DESCRIPTION
**Pull request by carpenterd777**

### CODE REVIEW

File: src/screens/DetailedViewScreen.tsx
Comment (aej11a) [inline src/screens/DetailedViewScreen.tsx:92]: This would be significantly improved by combining the labels and a11yDataList from above into one object, basically a map, and then loop through `Object.entries(<thatMap>)`

**Review by aej11a — CHANGES_REQUESTED** [review]

**Comment by carpenterd777** [comment]
Since main was merged into #216, I will wait until #216 is merged into main before starting to work on this branch again

**Comment by elenirotsides** [comment]
You should be good to go now @carpenterd777

**Comment by elenirotsides** [comment]
I think this is good but it doesn't seem to flow well (appearance-wise) with the other text on the page. The ratings are centered while the other stuff isn't, it just doesn't flow well for my liking. I'm wondering though if styling is out of scope for this issue? If so, lmk and we can make an issue for that. Figured I'd point this out though

IMG_0450

**Comment by carpenterd777** [comment]
> I think this is good but it doesn't seem to flow well (appearance-wise) with the other text on the page. The ratings are centered while the other stuff isn't, it just doesn't flow well for my liking. I'm wondering though if styling is out of scope for this issue? If so, lmk and we can make an issue for that. Figured I'd point this out though

yeah, you're right, the styling could be improved. compared to the UI on Yelp we could certainly afford to improve the margins on the text, perhaps add some more stylistic contrast to the address text compared to the title, and perhaps center elements that are not pure text (such as buttons). however, that would be a large adjustment, and I think we should make the call to add that in a separate issue. since this styling does not look great anyway, I'll just remove it for now.

I'll add a screenshot of the UI I was comparing it to below.

**Comment by elenirotsides** [comment]
> yeah, you're right, the styling could be improved. compared to the UI on Yelp we could certainly afford to improve the margins on the text, perhaps add some more stylistic contrast to the address text compared to the title, and perhaps center elements that are not pure text (such as buttons). however, that would be a large adjustment, and I think we should make the call to add that in a separate issue. since this styling does not look great anyway, I'll just remove it for now.

Yup I totally agree. I'll go ahead and open an issue for this

**Review by elenirotsides — APPROVED** [review]
Approving this, styling will be done in https://github.com/Peer-Stevens/peer/issues/237

**Comment by aej11a** [comment]
Agreed, we should definitely work on the styling, throughout the app

**Review by aej11a — APPROVED** [review]

**Pull request merged**
